blocks/extra-large-promo-block/features/extra-large-promo/default.jsx

```diff
@@ -222,7 +221,7 @@ const ExtraLargePromo = ({ customFields }) => {
 							</HeadingSection>
 						) : null}
 						{embedMarkup || showImage ? (
-							<MediaItem {...searchableField(imageSearchField)} suppressContentEditableWarning>
+							<MediaItem {...searchableField("imageOverrideURL")} suppressContentEditableWarning>
```

Comment:
Can we remove `searchableField` on line 197 too

Reply:
Thanks Matthew, removed [6a91f96](https://github.com/WPMedia/arc-themes-blocks/pull/1511/commits/6a91f96198b19c441978c6cc6a9f965593cd9c6b)